controllers/workshop_controller.go

```diff
@@ -242,3 +244,15 @@ func (r *WorkshopReconciler) SetupWithManager(mgr ctrl.Manager) error {
 		For(&workshopv1.Workshop{}).
 		Complete(r)
 }
+
+func (r *WorkshopReconciler) handleDelete(ctx context.Context, req ctrl.Request, workshop *workshopv1.Workshop) (ctrl.Result, error) {
+	log := r.Log.WithValues("workshop", req.NamespacedName)
+	log.Info("Deleting workshop" + workshop.ObjectMeta.Name)
+
+	giteaNamespaceName := "gitea"
```

Comment:
@MaazMS we shouldn't send namespace name from here! makes no sense. Lets rather define static constants in gitea controller itself

we shouldn't be leaking stuff like this! it's bad approach

Reply:
@rasheedamir  i will create constant for giteaNamespace Name in gitea controller